mockup/patterns/thememapper/js/lessbuilderview.js

```diff
@@ -52,10 +52,22 @@ define([
     },
     setFilename: function() {
         var self = this;
+
         if( self.app.lessPaths['save'] === undefined ) {
             return;
         }
-        var f = self.app.lessPaths['save'];
+
+        var filePath = self.app.lessPaths['less'];
+        var devPath = self.app.devPath[0];
```

Comment:
Yep, it will just change the current filename from .less to .css
On Aug 21, 2015 4:44 PM, "Nathan Van Gheem" <notifications@github.com>
wrote:

> In mockup/patterns/thememapper/js/lessbuilderview.js
> <https://github.com/plone/mockup/pull/558#discussion_r37681321>:
>
> >          if( self.app.lessPaths['save'] === undefined ) {
> >              return;
> >          }
> > -        var f = self.app.lessPaths['save'];
> > +
> > +        var filePath = self.app.lessPaths['less'];
> > +        var devPath = self.app.devPath[0];
>
> Will this be okay if no dev path is set in the manifest?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/plone/mockup/pull/558/files#r37681321>.
>